compiler/luci/pass/include/luci/Pass/QuantizeWithMinMaxPass.h

```diff
@@ -45,9 +45,10 @@ class QuantizeWithMinMaxPass : public logo::Pass
 public:
   QuantizeWithMinMaxPass(loco::DataType input_model_dtype, loco::DataType output_model_dtype,
                          QuantizationGranularity granularity, loco::DataType input_type,
-                         loco::DataType output_type)
+                         loco::DataType output_type, bool TF_style_maxpool)
```

Comment:
`TF_style_maxpool` is added but I counld search 'maxpool` in the issue and in the change message. Is this parameter necessary for reordering quantization steps?
Why I asked before and now is that from previous change I see (1) this introduction and (2) reorder of steps, which it would be better to split.

Reply:
Ah, I get the point.

Before this change, `TF_style_maxpool` was used by `PropagateQuantParamPass`, which is invoked after `QuantizeWithMinMaxPass` is called. This PR makes `PropagateQuantParamPass` be called inside `QuantizeWithMinMaxPass`, so the variable is moved together.

So, `TF_style_maxpool` is moved, not newly added.

Reply:
OK got it :)